src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -266,6 +267,13 @@ public void respawn()
 		alive = true;
 	}
 
+	public void moveInstantly(Location moveLocation) {
+		PlayerMoveEvent event = new PlayerMoveEvent(this, this.getLocation(), moveLocation);
+		Bukkit.getPluginManager().callEvent(event);
+		if(event.isCancelled() == false)
+			this.setLocation(moveLocation.clone());
```

Comment:
The `PlayerMoveEvent` can manipulate the destination, so we absolutely need to respect this modification.

```suggestion
			this.setLocation(event.getTo());
```

Reply:
I really didn't think about it, big brain!